run-tests.sh

```diff
@@ -7,8 +7,15 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle invenio_records_ui tests docs && \
-isort invenio_records_ui tests --check-only --diff && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+pydocstyle invenio_records_ui tests docs
+isort invenio_records_ui tests --check-only --diff
+check-manifest
+sphinx-build -qnNW docs docs/_build/html
+docker-services-cli up ${DB}
 python setup.py test
```

Comment:
Major: Missing the `docker-services-cli down` see https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/e3ca55f18a3e35ed4d3f5875d9647d408c45295b/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/run-tests.sh#L12-L14